docs/administration/crud-controller/reference/actions.md

```diff
@@ -0,0 +1,140 @@
+# Actions
+
+Actions provide a simple way for administrators to navigate the administration. You can use them for basic navigation between crud controllers, pages, or your custom controllers and actions.
+
+Actions can be configured in `configureActions()` method of the Crud Controller.
+
+```php
+namespace App\Controller\Admin;
+
+use App\Model\Article\Article;
+use Shopsys\AdministrationBundle\Component\Attributes\CrudController;
+use Shopsys\AdministrationBundle\Component\Config\Action\ActionsConfig;
+use Shopsys\AdministrationBundle\Controller\AbstractCrudController;
+
+#[CrudController(Article::class)]
+class ArticleController extends AbstractCrudController
+{
+    // ...
+
+    protected function configureActions(ActionsConfig $actions): ActionsConfig
+    {
+        
+    }
+}
+```
+
+## Default actions
+
+The Crud Controller provides a set of default actions that are displayed based on the Crud Controller configuration and specific page.
+
+!!! note
+
+    Default actions are displayed only if corresponding conditions are met (e.g. action is enabled).
+
+
+### New action
+
+- **Application Name**   : `create`
+- **Action**             : Redirects to the `ActionType::CREATE` of the current Crud Controller.
+- **Displayed on**       : `ActionType::LIST` page
+
+### Back to list action
+
+- **Application Name**   : `backToList`
+- **Action**             : Redirects to the `ActionType::LIST` of the current Crud Controller.
+- **Displayed on**       : `ActionType::DETAIL`, `ActionType::CREATE` and `ActionType::EDIT` pages
+
+
+## Custom actions
+
+You can add, update or remove actions in the `configureActions()` method. 
+
+```php
+// ...
+
+use App\Model\Article\Article;
+use Shopsys\AdministrationBundle\Component\Config\Action\ActionsConfig;
+use Shopsys\AdministrationBundle\Component\Config\Action\Builder\AbstractAction;
+use Shopsys\AdministrationBundle\Component\Config\Action\Builder\Action;
+use Shopsys\AdministrationBundle\Component\Config\ActionType;
+
+// ...
```

Comment:
in which class is this supposed to be?

Reply:
On top of this file is mentioned this:

Actions can be configured in `configureActions()` method of the Crud Controller.

```php
namespace App\Controller\Admin;

use App\Model\Article\Article;
use Shopsys\AdministrationBundle\Component\Attributes\CrudController;
use Shopsys\AdministrationBundle\Component\Config\Action\ActionsConfig;
use Shopsys\AdministrationBundle\Controller\AbstractCrudController;

#[CrudController(Article::class)]
class ArticleController extends AbstractCrudController
{
    // ...

    protected function configureActions(ActionsConfig $actions): ActionsConfig
    {
        
    }
}
```

So I think it's not necessary to specify everything in that sample :D

Reply:
you are right :+1: